isovar/cli/variant_sequences.py

```diff
@@ -80,9 +88,10 @@ def variant_sequences_generator_from_args(args):
     allele_reads_generator = allele_reads_generator_from_args(args)
     return reads_generator_to_sequences_generator(
         allele_reads_generator,
-        min_reads_supporting_cdna_sequence=args.min_reads_supporting_variant_sequence,
+        min_alt_rna_reads=args.min_alt_rna_reads,
+        min_variant_sequence_coverage=args.min_variant_sequence_coverage,
         preferred_sequence_length=args.cdna_sequence_length,
-        variant_cdna_sequence_assembly=args.variant_sequence_assembly)
+        variant_cdna_sequence_assembly=not args.disable_variant_sequence_assembly)
```

Comment:
you renamed this parameter to `variant_sequence_assembly` - think this should break now?